Formula/cqlkit.rb

```diff
@@ -0,0 +1,26 @@
+class Cqlkit < Formula
+  desc "CLI tool to export Cassandra query as CSV and JSON format"
+  homepage "https://github.com/tenmax/cqlkit"
+  url "https://github.com/tenmax/cqlkit/releases/download/v0.3.1/cqlkit-0.3.1.zip"
+  sha256 "f1d0cdbfa0780ee66c3b6e42d9eef1dd6b9c925918090ea2d4a547cb80dca4e8"
+
+  bottle :unneeded
+
+  depends_on :java => "1.8"
+
+  def install
+    libexec.install %w[bin lib]
+    bin.install_symlink libexec+"bin/cql2cql"
+    bin.install_symlink libexec+"bin/cql2csv"
+    bin.install_symlink libexec+"bin/cql2json"
```

Comment:
https://github.com/Homebrew/homebrew-core/blob/master/Formula/clojure.rb#L16 something like this would be better since it'll allow you to use the latest openjdk as a dependency.

Reply:
ok